### DISCUSSION TITLE
realted account active status

### DISCUSSION THREAD

**Discussion by rohithreddy1818** [community/community#143135] [GitHub Education]
### Hi everyone,

approved Submitted Oct 23, 2024
Congrats! Your request for [@rohithreddy1818](https://github.com/rohithreddy1818) was approved and your benefits will be available [soon](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits).still i didnt received my confirmation yet it has already 7 days passed.

**Comment by rohithreddy1818** [comment]
In my github student  its showing my status approved and it has been passed 7 days of time period too. still i didnt access to my benefits yet.can you please check it out .
![Screenshot 2024-10-30 205518](https://github.com/user-attachments/assets/7dcf88d1-199b-4aaf-833f-90f71f435d12)

**Comment by davevad93** [comment]
Hi @rohithreddy1818 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @rohithreddy1818 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
